src/components/table-of-contents/example-table-of-contents-sticky-full-page.njk

```diff
@@ -0,0 +1,114 @@
+---
+title: "Example: Table of contents on full page"
+layout: ~
+---
+{% from "components/table-of-contents/_macro.njk" import onsTableOfContents %}
+
+{% extends "layout/_template.njk" %}
+
+{% set pageConfig = {
+    "header": {
+        "title": "What is the census?"
+    },
+    "breadcrumbs": {
+        "ariaLabel": 'Back',
+        "itemsList": [
+            {
+                "url": '/',
+                "id": "back-example-feedback-form-error",
+                "text": 'Back',
+                "attributes": {
+                    "data-attribute": "Example attribute"
+                }
+            }
+        ]
+    },
+    "footer": {
+        "OGLLink": {
+            "pre": 'All content is available under the',
+            "link": 'Open Government Licence v3.0',
+            "url": 'https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/',
+            "post": ', except where otherwise stated'
+        }
+    }
+} %}
+
+{% block main %}
+    <div class="ons-page__container ons-container">
+        <div class="ons-grid ons-js-toc-container">
+            <div class="ons-grid__col ons-grid__col--sticky@m ons-col-4@m">
+                {{
+                    onsTableOfContents({
+                        "title": 'Contents',
+                        "ariaLabel": 'Sections in this page',
+                        "itemsList": [
+                            {
+                                "url": '#section1',
+                                "text": 'What is the benson?'
```

Comment:
benson?

Reply:
Good spotting. Needed a change of title and accidentally left it there. Was probably thinking about sleeping and benson for beds :)